Bolts/Common/BFTask.h

```diff
@@ -83,6 +83,14 @@ typedef __nullable id(^BFContinuationBlock)(BFTask<ResultType> *task);
 + (instancetype)taskForCompletionOfAllTasksWithResults:(nullable NSArray<BFTask *> *)tasks;
 
 /*!
+ Returns a task that will be completed once there is at least one successful task.
+ The first task to successuly complete will set the result, all other tasks results are 
+ ignored.
```

Comment:
Maybe expand on behavior of errors/exceptions/cancellation, as without documentation it's an implementation detail at this point.